xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/editors/dataeditors.css

```diff
@@ -323,20 +323,21 @@ div#xwikiobjects input[type="radio"], div#xwikiclassproperties input[type="radio
   text-decoration: underline;
   background-color: $theme.highlightColor;
 }
-
+#add_xobject .buttonwrapper input , #add_xproperty .buttonwrapper input {
```

Comment:
The best practice is to put each selector on a separate line. The comma ``,`` is hard t spot and you can easily be mislead that it's a single long selector.